src/build.config.js

```diff
@@ -18,7 +18,8 @@ export default {
       'PWETH',
       'ARBETH',
       'FISH',
-      'SOL'
+      'SOL',
+      'FUSE'
     ],
     testnet: [
       'BTC',
```

Comment:
Could you also add fuse to testnet so we can test it ?

Reply:
> I think this causes not to be able to use fuse

how did you get to this error? I would like to simulate it.